components/dashboard/package.json

```diff
@@ -14,39 +14,38 @@
     "xterm-addon-fit": "^0.5.0"
   },
   "devDependencies": {
-    "@tailwindcss/forms": "^0.2.1",
-    "@tailwindcss/postcss7-compat": "^2.0.2",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
+    "@craco/craco": "^6.1.1",
+    "@tailwindcss/forms": "^0.3.2",
+    "@tailwindcss/postcss7-compat": "^2.1.0",
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.1.10",
     "@types/classnames": "^2.2.11",
-    "@types/jest": "^26.0.15",
+    "@types/jest": "^26.0.22",
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "@types/react-router": "^5.1.12",
+    "@types/react-dom": "^17.0.3",
+    "@types/react-router": "^5.1.13",
     "@types/react-router-dom": "^5.1.7",
-    "@typescript-eslint/eslint-plugin": "^4.15.2",
-    "@typescript-eslint/parser": "^4.15.2",
+    "@typescript-eslint/eslint-plugin": "^4.21.0",
+    "@typescript-eslint/parser": "^4.21.0",
     "autoprefixer": "^9.8.6",
-    "classnames": "^2.2.6",
-    "eslint": "^7.21.0",
+    "classnames": "^2.3.1",
+    "eslint": "^7.23.0",
```

Comment:
it would be good to use a single version per component.

```
gitpod /workspace/gitpod $ yarn why eslint
yarn why v1.22.10
[1/4] Why do we have the module "eslint"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "eslint@7.24.0"
info Reasons this module exists
   - "_project_#@gitpod#dashboard" depends on it
   - Hoisted from "_project_#@gitpod#dashboard#eslint"
info Disk size without dependencies: "9.66MB"
info Disk size with unique dependencies: "28.98MB"
info Disk size with transitive dependencies: "33.41MB"
info Number of shared dependencies: 87
=> Found "react-scripts#eslint@7.21.0"
info This module exists because "_project_#@gitpod#dashboard#react-scripts" depends on it.
```

Reply:
fixed